docs/network/smart-rollup-nodes.md

```diff
@@ -18,65 +18,92 @@ Make sure that you understand the interaction between different nodes as describ
 
 For more information about Smart Rollup nodes in general, see [Smart Rollups](https://docs.tezos.com/architecture/smart-rollups) on docs.tezos.com and [Smart Rollup Node](https://tezos.gitlab.io/shell/smart_rollup_node.html) in the Octez documentation.
 
-## Running the Smart Rollup node from a snapshot
+You can run the Etherlink Smart Rollup in many modes, but these instructions cover only the most common modes:
 
-You can start the Smart Rollup node with a snapshot of the Etherlink state to prevent it from having to compute the state from Etherlink genesis.
+- [Running the Smart Rollup node in operator mode](#running-the-smart-rollup-node-in-operator-mode)
+- [Running the Smart Rollup node in observer mode](#running-the-smart-rollup-node-in-observer-mode)
 
-For simplicity, these steps show how to run the Smart Rollup node in observer mode:
+In either case, you can start the Smart Rollup node with a snapshot of the Etherlink state to prevent it from having to compute the state from Etherlink genesis.
+You can also skip the snapshot and run the node from Etherlink genesis, but it can take a long time for the node to catch up.
+These instructions cover both ways of running the Smart Rollup node.
+
+## Running the Smart Rollup node in operator mode
+
+Running the Smart Rollup node in operator mode is the best way to participate in Etherlink because in this mode the node posts commitments about Etherlink's state to layer 1.
+However, running the node in this mode requires an account with at least 10,000 staked tez and certain other prerequisites.
+
+To run an Etherlink Smart Rollup node in operator mode, you need:
+
+- A Tezos layer 1 node running in archive mode or full mode with expanded node history, such as with the argument `--history-mode full:50`
+- An account with at least 10,000 tez staked, referred to as the _operator account_
+- A separate account to sign batching operations, referred to as the _batcher account_; this account is necessary only if you also run an Etherlink EVM node
```

Comment:
This is actually not true. Only the sequencer needs a batcher key.

Maybe we can start pushing for the “maintenance” mode for operators.

Reply:
Removed mention of batcher key.